File: solutions/csharp/lucians-luscious-lasagna/3/LuciansLusciousLasagna.cs
class Lasagna
{
    public int ExpectedMinutesInOven() => 40;
    public int RemainingMinutesInOven(int actual) => ExpectedMinutesInOven() - actual;
    public int PreparationTimeInMinutes(int layers) => layers * 2;
    public int ElapsedTimeInMinutes(int layers, int minutes) => PreparationTimeInMinutes(layers) + minutes;
}

File: solutions/fsharp/lucians-luscious-lasagna/1/LuciansLusciousLasagna.fs
module LuciansLusciousLasagna

let expectedMinutesInOven = 40

let remainingMinutesInOven m = expectedMinutesInOven - m 

let preparationTimeInMinutes layers = layers * 2

let elapsedTimeInMinutes layers minutes = 
    let p = preparationTimeInMinutes layers
    p + minutes
